backoffice/templates/backoffice/vehicles/details.html

```diff
@@ -0,0 +1,96 @@
+{% extends "backoffice_base.html" %}
+
+{% load static %}
+{% block scripts %}
+<script type="module" src="{% static "get-telemetry.js" %}" defer></script>
+{% endblock %}
+
+{% block heading %}<h2 class="bg-yellow-400 text-black font-bold text-xl px-6 py-2 rounded-md border-4 border-black tracking-widest shadow-md w-fit">{{ vehicle.registration }}</h2>{% endblock %}
+{% block buttons %}
+
+{% endblock %}
+
+{% block main %}
+
+<a class="hidden" href="{{ request.get_host }}" id="base_url"></a>
+<a href="{% url "get_telemetry" %}" class="hidden" id="telemetry_url"></a>
+<input type="hidden" id="vehicle_id" value="{{ vehicle.id }}">
+{% csrf_token %}
+
+<div class="m-8 flex flex-row gap-2">
+    <img src="{% if vehicle.picture %}{{ vehicle.picture.url }}{% endif %}""><div class="relative w-32 h-32">
+    <svg class="w-full h-full transform -rotate-90" viewBox="0 0 100 100">
+        <circle class="text-gray-200 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent" />
+{% if most_recent.soc.value > 75 %}
+        <circle class="text-green-500 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent"
+stroke-dasharray="251.2" stroke-dashoffset="{{ most_recent.soc_dial }}" stroke-linecap="round" />
+{% elif most_recent.soc.value > 50 %}
+        <circle class="text-amber-500 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent"
+stroke-dasharray="251.2" stroke-dashoffset="{{ most_recent.soc_dial }}" stroke-linecap="round" />
+{% else %}
+        <circle class="text-red-500 stroke-current" cx="50" cy="50" r="40" stroke-width="10" fill="transparent"
+stroke-dasharray="251.2" stroke-dashoffset="{{ most_recent.soc_dial }}" stroke-linecap="round" />
+{% endif %}
+    </svg>
+    <div class="absolute inset-0 flex items-center justify-center text-xl font-bold text-gray-800">
+      {{ most_recent.soc.value }}%
```

Comment:
We also need to clearly show the date of the last soc value (and ideally text for how long ago that was as well), because otherwise we can't easily see how out of date the information is - and it's really important to be able to see that at a glance from practical experience of using the state of charge info on the dashboard screen.